### PR TITLE
Feat(wallet):  renaming `wallet` notifications to `account`

### PR DESCRIPTION
**Pull request by cmonti-bc**
## Description (optional)
On apps unification a single account will be created, so `wallet` naming should be changed into `account`.

## Testing Steps (optional)
Testing should be done by creating a new account and checking notifications and site's templates for right or wrong account creation.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx [hunk 1]
```diff
@@ -504,8 +504,8 @@ const getAlertContent = (message, data = undefined) => {
     case C.MOBILE_LOGIN_ERROR:
       return buildMessageTemplate(
         <FormattedMessage
-          id='components.alerts.mobile_login_error'
-          defaultMessage='Error logging into your wallet'
+          id='components.alerts.mobile_login_account_error'
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx:507]: Nice work! You'll also need to update the related strings in `blockchain-wallet-v4-frontend/packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts`

So for example, in that file you'll see an entry for `components.alerts.mobile_login_error`. Just change that text the same as you did in the template files. You'll need to do this for each of the text strings you've changed in this PR.

**Review by milan-bc — CHANGES_REQUESTED** [review]
Great work so far, please follow up @blockdylanb comment and add new label inside packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts 👍

File: packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx [hunk 2]
```diff
@@ -1013,8 +1013,8 @@ const getAlertContent = (message, data = undefined) => {
     case C.WALLET_UPGRADE_ERROR:
       return buildMessageTemplate(
         <FormattedMessage
-          id='components.alerts.wallet_upgrade_error'
-          defaultMessage='Failed to upgrade to HD and save wallet.'
+          id='components.alerts.account_upgrade_error'
+          defaultMessage='Failed to upgrade to HD and save account.'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx:1017]: `Failed to upgrade account to HD wallet`

File: packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmailToken/template.success.tsx [hunk 3]
```diff
@@ -27,14 +27,14 @@ const Success = () => {
         <Text color='grey900' style={{ marginTop: '8px' }} size='16px' weight={500}>
           <FormattedMessage
             id='scenes.verifyemailtoken.return_mobile'
-            defaultMessage='Return to the app to access your Blockchain Wallet.'
+            defaultMessage='Return to the app to access your Blockchain Account.'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmailToken/template.success.tsx:30]: `Return to the app to access your Blockchain.com Account.`

File: packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmailToken/template.success.tsx [hunk 4]
```diff
@@ -27,14 +27,14 @@ const Success = () => {
...
           />
         </Text>
       ) : (
         <Text color='grey900' style={{ marginTop: '8px' }} size='16px' weight={500}>
           <FormattedMessage
             id='scenes.verifyemailtoken.return'
-            defaultMessage='Return to the previous tab to access your Blockchain Wallet.'
+            defaultMessage='Return to the previous tab to access your Blockchain Account.'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmailToken/template.success.tsx:37]: `Return to the previous tab to access your Blockchain.com Account.`

**Review by schnogz — CHANGES_REQUESTED** [review]

**Comment by cmonti-bc** [comment]
> Great work so far, please follow up @blockdylanb comment and add new label inside packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts 👍

Finally I didn't create any new label, I modified the ones that were used.

**Review by blockdylanb — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx [hunk 5]
```diff
@@ -1013,8 +1013,8 @@ const getAlertContent = (message, data = undefined) => {
     case C.WALLET_UPGRADE_ERROR:
       return buildMessageTemplate(
         <FormattedMessage
-          id='components.alerts.wallet_upgrade_error'
-          defaultMessage='Failed to upgrade to HD and save wallet.'
+          id='components.alerts.account_upgrade_error'
```
Comment (amadeo-bc) [inline packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx:1016]: @cmonti-bc you haven't changed the key for this so it will fail.

**Review by amadeo-bc — CHANGES_REQUESTED** [review]

File: packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx (hunk 5 above)
Comment (cmonti-bc) [reply packages/blockchain-wallet-v4-frontend/src/components/Alerts/messages.tsx:1016]: great catch!

**Review by schnogz — APPROVED** [review]

**Pull request merged**